src/content/reference/react-dom/hooks/useFormStatus.md

```diff
@@ -0,0 +1,259 @@
+---
+title: useFormStatus
+canary: true
+---
+
+<Canary>
+
+The `useFormStatus` Hook is currently only available in React's canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+<Intro>
+
+`useFormStatus` is a Hook that gives you status information of the last form submission.
+
+```js
+const { pending, data, method, action } = useFormStatus();
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `useFormStatus()` {/*use-form-status*/}
+
+The `useFormStatus` Hook provides status information of the last form submission.
+
+```js [[1, 5, "status.pending"]]
+import action from './actions';
+
+function Submit() {
+  const status = useFormStatus();
```

Comment:
nit: should we use the same convention as earlier?
`const { pending } = useFormStatus();`

Reply:
I intentionally replaced this with the object as I wanted to mirror the definition that `useFormStatus` returns status information, more than just `pending`. 

True, it's cleaner to use `{pending}` but my impression is that this version puts less emphasis on the other properties that are returned in `status` -- but just slightly. So happy to change if you strongly disagree.